lib/semian/circuit_breaker.rb

```diff
@@ -37,33 +37,33 @@ def acquire(resource = nil, &block)
       result
     end
 
-    def half_open?
```

Comment:
This change reverts https://github.com/Shopify/semian/pull/189, is that what you wanted?